docs/.gitignore

```diff
@@ -0,0 +1,5 @@
+_site
+.sass-cache
+.jekyll-metadata
+Gemfile
```

Comment:
Is there a reason we are ignoring the Gemfiles? This makes it a bit difficult for other people to setup and run locally since the tutorials all try to get you to run `jekyll new` commands (which fail on a non-empty directory) to create the Gemfile.

Reply:
Unlike regular RoR apps, it appears GitHub pages totally ignores the gemfile and bundler gemfile.lock anyway, so it's fine to check them in to speed up local development.